### PR TITLE
Add note/faq about speeding up regression.learn.CleanLearning runtime

### PR DESCRIPTION
**Pull request by huiwengoh**

### CODE REVIEW

File: cleanlab/regression/learn.py [hunk 1]
```diff
@@ -127,6 +127,7 @@ class CleanLearning(BaseEstimator):
     n_boot :
         Number of bootstrap resampling rounds used to estimate the model's epistemic uncertainty.
         Default is 5. Larger values are expected to produce better results but require longer runtimes.
+        Set as 0 to skip estimating the epistemic uncertainty.
```
Comment (jwmueller) [inline cleanlab/regression/learn.py:130]:
```suggestion
        Set as 0 to skip estimating the epistemic uncertainty and get results faster.
```

File: docs/source/tutorials/faq.ipynb [hunk 2]
```diff
@@ -550,6 +550,55 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "623df36d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from cleanlab.regression.learn import CleanLearning\n",
+    "\n",
+    "X = np.random.random(size=(200, 3))\n",
```
Comment (jwmueller) [inline docs/source/tutorials/faq.ipynb:590]:
```suggestion
    "X = np.random.random(size=(30, 3))\n",
```

Comment (jwmueller) [reply docs/source/tutorials/faq.ipynb:590]: will 30 work, or do we need bigger value like 200?

Comment (jwmueller) [reply docs/source/tutorials/faq.ipynb:590]: prefer these run instantly

File: docs/source/tutorials/faq.ipynb [hunk 3]
```diff
@@ -550,6 +550,55 @@
...
+    "\n",
+    "- Setting `include_aleatoric_uncertainty=False` to skip the aleatoric uncertainty estimation.\n",
+    "\n",
+    "- Having less elements in the `coarse_search_range` of [regression.learn.CleanLearning.find_label_issues](../cleanlab/regression/learn.html#cleanlab.regression.learn.CleanLearning.find_label_issues). This is the coarse search range to find the value of `k`, which estimates the fraction of data that have label issues. More values represent a more thorough search, hence we can reduce the runtime of this method by reducing this search range.\n",
+    "\n",
+    "- Reducing the `fine_search_size` in [regression.learn.CleanLearning.find_label_issues](../cleanlab/regression/learn.html#cleanlab.regression.learn.CleanLearning.find_label_issues). This is the size of the fine-grained search grid to find the value of `k`, which estimates the fraction of data that have label issues. A higher number represent a more thorough search, hence we can reduce the runtime of this method by using a smaller search size.\n",
```
Comment (jwmueller) [inline docs/source/tutorials/faq.ipynb:576]:
```suggestion
    "To effectively identify errors in a regression dataset, the methods in [regression.learn.CleanLearning](../../cleanlab/regression/learn.html#cleanlab.regression.learn.CleanLearning) estimate each datapoint's aleatoric uncertainty (by fitting a second copy of the regression model to predict the residuals’ magnitudes), as well as its epistemic uncertainty (by fitting multiple copies of the regression model with bootstrap resampling). These uncertainty estimates help provide a robust quality score that accounts for the model's imperfect predictions. \n",
    "\n",
    "These uncertainty estimates help produce better results but require longer runtimes. Here are a few options to speed up the runtime of these methods:\n",
    "\n",
    "- Reduce the number of bootstrap resampling rounds by decreasing the `n_boot` argument (default value is 5, set it to 0 to skip the epistemic uncertainty estimation entirely).\n",
    "\n",
    "- Set `include_aleatoric_uncertainty=False` to skip the aleatoric uncertainty estimation.\n",
    "\n",
    "- Include less elements in the `coarse_search_range` argument of [regression.learn.CleanLearning.find_label_issues](../cleanlab/regression/learn.html#cleanlab.regression.learn.CleanLearning.find_label_issues). This is overall set of values initially considered for estimating the fraction of data that have label issues.\n",
    "\n",
    "- Reduce the `fine_search_size` argument of  [regression.learn.CleanLearning.find_label_issues](../cleanlab/regression/learn.html#cleanlab.regression.learn.CleanLearning.find_label_issues). A higher number represents a more thorough search to precisely estimate the fraction of data that have label issues.\n",
```

**Review by jwmueller — APPROVED** [review]
LGTM! just left some some suggestions you can add or skip, and then can go ahead and merge!

File: docs/source/tutorials/faq.ipynb (hunk 2 above)
Comment (huiwengoh) [reply docs/source/tutorials/faq.ipynb:590]: Yup 30 works, will change this

**Pull request merged**